files/disable_inactive_freeipa_users.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/bash
+
+# Usage:
+#   disable_inactive_freeipa_users.sh [disable_time]
+#
+# A user is considered inactive if he or she has not logged since
+# disable_time.  If disable_time is not specified then a default value
+# of "45 days ago" is used.
+#
+# The date/time must be specified so that it understandable by the
+# date command, which is the same as saying it must be understandable
+# by coreutils.  See the "Date input formats" section of the info
+# manual distributed with coreutils for more information on date/time
+# formats:
+# https://www.gnu.org/software/coreutils/manual/html_node/Date-input-formats.html
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# A user is considered inactive if he or she has not logged since this
```

Comment:
Use more inclusive wording and add a missing word:
```suggestion
# A user is considered inactive if they have not logged in since this
```

Reply:
See [8d971f1](https://github.com/cisagov/ansible-role-freeipa-server/pull/73/commits/8d971f1de83ce7394cb916e1fc1e2ce418671208) for the missing word.

Reply:
Please see commit [43c6240](https://github.com/cisagov/ansible-role-freeipa-server/pull/73/commits/43c6240629a59d716428f7602f86189967251f81).